aste-turbine/run.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+# This script assumes the ASTE binaries and python scripts are in $PATH
+
+# Download the meshes
+test -f meshes.tar.gz  || wget https://gitlab.lrz.de/precice/precice2-ref-paper-setup/-/raw/main/meshes/meshes.tar.gz
+
+# Extract the meshes
+tar -xvf meshes.tar.gz
```

Comment:
You can even remove the verbosity here, it makes output a bit more difficult to parse:
```suggestion
tar -xf meshes.tar.gz
```